### PR TITLE
RSK Mainnet configuration

### LINKED ISSUE #1990: RSK Mainnet network
### Description of the Feature

Addition of RSK Mainnet network configuration

### PR DESCRIPTION
**Pull request by alepc253**
Closes #1990

### Description

Configuration of RSK Mainnet network

### Changes

* Basic config
* Checksum unit testing with mainnet address

### Steps to Test

1. Connect to RSK network
2. Navigate to Contracts tab
3. Execute the only method of Bridge contract and validate you get a response

### CODE REVIEW

**Comment by alepc253** [comment]
Hi @dternyak, here it is the configuration for RSK mainnet, with a dedicated node.

File: common/containers/Tabs/Contracts/index.tsx [hunk 1]
```diff
@@ -27,7 +27,13 @@ class Contracts extends Component<RouteComponentProps<{}>> {
 
     return (
       <TabSection isUnavailableOffline={true}>
-        <SubTabs tabs={tabs} match={match} location={location} history={history} />
+        <SubTabs
+          tabs={tabs}
+          match={match}
+          location={location}
+          history={history}
+          staticContext={this.props.staticContext}
```
Comment (wbobeirne) [inline common/containers/Tabs/Contracts/index.tsx]: This addition of staticContext should be unnecessary, make sure you installed the type definitions using `yarn` as this looks like a typedef / library version mismatch if it's warning you about this.

Comment (alepc253) [reply common/containers/Tabs/Contracts/index.tsx]: @wbobeirne ok, I'll that. This was just because I got an error and couldn't push

Comment (alepc253) [reply common/containers/Tabs/Contracts/index.tsx]: @wbobeirne fixed

**Comment by dternyak** [comment]
Hi @alepc253. 

I'll be merging this PR, and it will go out with our next release. We generally make releases every 2-3 weeks, so you should expect one soon™ 😁 

We're going to be wrapping up the current release shortly, and then guiding it towards production over the next week(s). 

I'll be sure to make you aware when this goes out to production!

**Pull request merged**